packages/calypso-e2e/src/lib/blocks/block-flows/instagram.ts

```diff
@@ -46,27 +44,39 @@ export class InstagramBlockFlow implements BlockFlow {
 		const embedButtonLocator = editorCanvas.locator( selectors.embedButton );
 		await embedButtonLocator.click();
 
-		// We should make sure the actual Iframe loads, because it takes a second.
-		const instagramIframeLocator = editorCanvas
-			.frameLocator( selectors.embedIframe )
-			.frameLocator( selectors.instagramIframe )
-			.locator( `text=${ this.configurationData.expectedPostText }` )
-			.first();
+		const response = {
+			version: '1.0',
+			author_name: 'woocommerce',
+			provider_name: 'Instagram',
+			provider_url: 'https://www.instagram.com/',
+			html: '<a href="">A post shared by WooCommerce (@woocommerce)</a>',
+		};
```

Comment:
Not sure if we should be stubbing the payload here. What if the actual payload changes? If that happens and the block implementation gets updated, we'll be seeing false positives because we have a deprecated response payload.

Reply:
Yes, that makes sense. Perhaps just stub HTTP 200 at most? 

I see in your implementation that you don't bother intercepting the request; that works too.

Reply:
Yeah, I guess we can leave that for the dedicated/upstream block tests. I think seeing that the block gets added and doesn't error out when submitting the embedded URL should be sufficient in our case.